draft/elm.md

```diff
@@ -0,0 +1,30 @@
+# Elm, functional frontend development
+
+## … and why you should care
+
+A few months ago I got to know about [Elm](http://elm-lang.org/), which claims to be "the best of functional programming in your browser". After inhaling the documentation and watching a few videos and talks, I started tinkering myself and quickly fell in love with [Elm, functional frontend development](https://dennisreimann.de/articles/elm.html), and the feeling it gives me when writing code in it.
+
+The last time I felt this excitement of working with something radically new was when I learned Rails – almost ten years ago. Back then one could feel that Rails would be a big shift, because it was a coherent package that bundled up web dev learnings, best practices, and some very nice ideas for its future development. Right now this is happening again with Elm, so let us have a look at what is exciting me …
+
+In 2015 functional (reactive) programming advanced in the frontend field and by now every frontend developer should have heard of it. Functional programming paradigms and ideas are influencing JavaScript and its frameworks: [PureScript](http://www.purescript.org/) and [RxJS](https://github.com/Reactive-Extensions/RxJS) are examples for languages and libraries that build on the functional-reactive concepts – [React](https://facebook.github.io/react/) and accompanying libraries are increasingly influenced by the functional trend as well.
+
+But instead of mounting a framework for working in a functional manner on top of our good old JavaScript, Elm is a whole new language that builds on functional paradigms and makes reactive programming a core concept. And even so it compiles to JavaScript so it can run in the browser, Elm does not try to bring functional programming to JavaScript, but to its developers.
```

Comment:
Let's split that last part up in to two sentences. Instead of:

> And even so it compiles to JavaScript so it can run in the browser, Elm does not try to bring functional programming to JavaScript, but to its developers.

Let's do:

> And even so, it compiles to JavaScript so it can run in the browser. Elm does not try to bring functional programming to JavaScript, but to its developers.